libraries/AdaptiveExpressions/parser/ExpressionParser.cs

```diff
@@ -275,6 +275,12 @@ public override Expression VisitStringInterpolationAtom([NotNull] ExpressionAntl
                     }
                 }
 
+                // If there is nothing in the string interpolation, add a empty string to the children list.
+                if (children.Count == 0)
```

Comment:
How about appending an empty string to all the children?
```
var children = new List<Expression>() { string.Empty };
```

Reply:
changed